### PR TITLE
bump up to rhamt 0.0.19; mark 0.0.11...

### PR DESCRIPTION
**Pull request by nickboldt**
bump up to rhamt 0.0.19; mark 0.0.11 deprecated

Change-Id: Ia2d8e669d20374068ef6863b01f1143739726349
Signed-off-by: nickboldt <nboldt@redhat.com>

### CODE REVIEW

**Comment by nickboldt** [comment]
this update also solves https://github.com/windup/rhamt-vscode-extension/issues/263

**Comment by nickboldt** [comment]
cc: @johnsteele FYI

File: v3/plugins/redhat/rhamt/0.0.19/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+apiVersion: v2
+id: redhat.rhamt-vscode-extension
```
Comment (amisevsk) [inline v3/plugins/redhat/rhamt/0.0.19/meta.yaml]: This `id` field should not be present; it breaks workspace startup.

File: v3/plugins/redhat/rhamt/0.0.19/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    targetPort: 61436
+    attributes:
+      protocol: http
+  - name: "report-endpoint"
+    public: true
+    targetPort: 61435
+    attributes:
+      protocol: http
+  containers:
+  - image: "docker.io/windup3/rhamt-vscode-extension:java8"
```
Comment (nickboldt) [inline v3/plugins/redhat/rhamt/0.0.19/meta.yaml]: @johnsteele how is this image built? if we want to include RHAMT ootb in CRW 2 we need to know how this sidecar is created and make a RH-compatible one.

**Comment by amisevsk** [comment]
This PR may be affected by the webviews issues that are only fixed in theia/next.

**Comment by nickboldt** [comment]
related downstream https://issues.redhat.com/browse/CRW-639

**Comment by nickboldt** [comment]
Hey @johnsteele can we get a witness? :D

**Comment by johnsteele** [comment]
@nickboldt I'm wondering if we could jump to version 0.0.23 for `rhamt-vscode-extension-0.0.23-427.vsix`. It was published today with the necessary changes for the latest version of VSCode. I also need to update our demo project (https://github.com/windup/rhamt-che-demo) with this version.

**Comment by nickboldt** [comment]
By all means. Do you want to edit this PR? or submit a new one? If the former, go for it. If the latter LMK and I'll close this one. (Or just close it for me.)

**Comment by nickboldt** [comment]
@johnsteele psst. what do you want to do here?

**Comment by johnsteele** [comment]
@nickboldt Sorry for late response here. We can close this. It's already been bumped to 0.0.23.

**Pull request closed without merging**